docker/docker.go

```diff
@@ -108,10 +108,8 @@ func ExecLogin(serverAddress, username, token string) error {
 		RegistryToken: token,
 	}
 
-	response, err = cli.RegistryLogin(ctx, types.AuthConfig(*authConfig))
-	if err != nil {
-		return errors.Errorf("error saving credentials: %v", err)
-	}
+	// TODO: @andriisoldatenko add proper error handling
+	response, _ = cli.RegistryLogin(ctx, types.AuthConfig(*authConfig))
```

Comment:
Should we print the error instead of reverting to the "disappearing" error?

Reply:
problem that `RegistryLogin ` doesn't work for a while i have a ticket to fix it.

Reply:
also printing this error, doesn't make any sense, we have to fix it in next release.

Reply:
Wouldn't printing it give us visibility into the error the CLI is seeing? (without effecting the code logic)

Reply:
Had we been printing errors, this issue may have been noticed prior to releasing a new minor version. Printing seems like the best option, so that if there is another issue, we at least get some insight into what is happening.

Reply:
i agree with you, but it will confuse users.
@ashb @schnie @paolaperaza wdyt?

Reply:
So long as we print it with something like "Registry log in may not work". Otherwise just printing an error is probably as confusing as not printing it at all

Reply:
https://github.com/astronomer/issues/issues/1249

Reply:
Hmm, what is the underlying error though? Seems like this error happens regularly and is almost expected? Printing a message every time does seem confusing unless I'm misunderstanding when this error would actually be shown.